service/awsconfig/v10/resource/loadbalancer/elb.go

```diff
@@ -16,20 +14,6 @@ const (
 	loadBalancerTagChunkSize     = 20
 )
 
-func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interface{}, error) {
-	customObject, err := key.ToCustomObject(obj)
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
-
-	currentState, err := r.clusterLoadBalancers(customObject)
-	if err != nil {
-		return nil, microerror.Mask(err)
-	}
-
-	return currentState, nil
-}
-
```

Comment:
I renamed `current.go` to `elb.go` and we no longer need the `GetCurrentState` method.